scripts/repo_build.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+"""Build all packages and create a package index."""
+
+import argparse
+import logging
+import sys
+import docker
+from toltec.builder import Builder
+from toltec.repo import Repo
+from toltec.util import argparse_add_verbose, LOGGING_FORMAT
+
+parser = argparse.ArgumentParser(description=sys.modules[__name__].__doc__)
```

Comment:
This can get shortened (same as in `package_build.py`)

```suggestion
parser = argparse.ArgumentParser(description=__doc__)
```